src/db.rs

```diff
@@ -335,5 +335,4 @@ CREATE table review_prefs (
 CREATE EXTENSION intarray;
 CREATE UNIQUE INDEX review_prefs_user_id ON review_prefs(user_id);
  ",
-    "ALTER TABLE review_prefs ADD COLUMN max_assigned_prs INTEGER DEFAULT NULL;",
```

Comment:
This looks iffy, but I think should be ok to revert. We'll need to be careful when reapplying this change.

Reply:
good point, I'll keep it in mind, thanks